docs/CONTRIBUTING.md

```diff
@@ -15,6 +15,12 @@ To set up a development environment, please follow these steps:
    cd starknet.quest
    npm i && npm run dev
    ```
+### 
+To avoid any failure to resolve the dependency tree, the --force flag should be added:
```

Comment:
This is not the aim of this issue, please remove any change made on this file (someone else is already working on it)